components/crypto/src/proto/mod.rs

```diff
@@ -14,8 +14,10 @@
 
 pub use crate::proto::schema::*;
 
-use exonum_proto::ProtobufConvert;
-use failure::{ensure, format_err, Error};
+use exonum_proto::{
+    failure::{ensure, format_err, Error},
```

Comment:
Relying on reexports that should logically be an implementation detail is a no-go, sorry.